packages/nodejs/src/diagnose.ts

```diff
@@ -93,15 +93,16 @@ export class DiagnoseTool {
 
     // we want to fall over if this value isn't present
     // (it should be)
-    const logPath = this.#config.data.logPath!
+    const logDirPath = this.#config.data.logPath!
+    const logPath = logDirPath + "/appsignal.log"
```

Comment:
We also have an (internal?) `logFilePath` config value: https://github.com/appsignal/appsignal-nodejs/blob/8a26df7974dbb2751acf561890118e4fb6d1812e/packages/nodejs/src/config.ts#L82-L86

Could we use that instead? I tried the code below but got a TypeScript error I don't quite understand yet.

```suggestion
    const logPath = this.#config.data.logFilePath!
```

```
src/diagnose.ts:135:9 - error TS2322: Type '{ exists: false; path: string; } | { exists: false; path: string; } | { exists: false; path: string | boolean | string[]; content: string[]; }' is not assignable to type 'FileMetadata'.
  Type '{ exists: false; path: string | boolean | string[]; content: string[]; }' is not assignable to type 'FileMetadata'.
    Types of property 'path' are incompatible.
      Type 'string | boolean | string[]' is not assignable to type 'string | undefined'.
        Type 'false' is not assignable to type 'string | undefined'.

135         paths[key] = {
```